### DISCUSSION TITLE
Xiaomi ZNCZ04LM fw 01-23-2022 issue with IKEA remotes.

### DISCUSSION THREAD

**Discussion by paulz-gh** [Koenkk/zigbee2mqtt#12623] [Devices]
Hi, 
I recently updated 2 of these "Mi power plug ZigBee EU" plugs by temporarily attaching them to Aqara G2H.
After I rejoined them to my Z2M network I noticed they switched on/off when I used my IKEA E1524/E1810 remotes to switch the lights they are attached to. When the light goes on the plug switches on but only when I use the IKEA remote, not when I switch them in the Z2M web gui.
Doing some investigation I noticed the IKEA remotes were using the plugs as their router. 
When I pulled the plugs and force the remotes to find another router the problem is gone. 

Could this be a bug in the new firmware ? Is it possible the plugs execute the command in addition to routing it ?

The remotes are bound to the lights using the IKEA method of pairing them by holding them close to the lights when resetting them. So the are not bound in Z2M. 

I tried to downgrade the firmware to the version available in the OTA index using a local index.json and the force flag but unfortunately the plugs refused to downgrade.

Could breaking the IKEA bind and using Z2M binding help ?

TIA
 Paul

**Comment by paulz-gh** [comment]
Removed the IKEA way of binding the remote and the lights and added a Z2M binding. Now the Xiaomi plugs no longer act on the commands sent by remote to the lights even when acting as router for the remote.
Seems like an acceptable solution to me

**Reply by rpimaster88** [reply]
How did you Do this. I have the same Problem after i updated my xiaomi plug via ota from Zigbee2mqtt. Now the Ikea remote controls the plug. How did you resolve this? Thanks

**Comment by paulz-gh** [comment]
﻿Hi, I am on vacation at the moment, so I cannot check but I think I removed the direct binding of the ikea remotes and the lights and replaced it with z2m binding.

Paul

**Reply by rpimaster88** [reply]
Hi, How did you remove it? And how did you bind them again? Did you stand directly to the zigbee Stick?

**Comment by paulz-gh** [comment]
I think I did a reset of the remotes and re-paired them with the z2m server. 
Then I bound the remotes to the lights in z2m. 

Regards,
 Paul
